frontend/packages/operator-lifecycle-manager/src/components/clusterserviceversion.tsx

```diff
@@ -849,15 +851,30 @@ export const ClusterServiceVersionsDetailsPage: React.FC<ClusterServiceVersionsD
       breadcrumbsFor={() => [
         {
           name: 'Installed Operators',
-          path: `/k8s/ns/${props.match.params.ns}/${props.match.params.plural}`,
+          path: `/k8s/ns/${namespace}/${props.match.params.plural}`,
         },
         { name: 'Operator Details', path: props.match.url },
       ]}
       resources={[
-        { kind: referenceForModel(SubscriptionModel), isList: true, prop: 'subscriptions' },
-        { kind: referenceForModel(PackageManifestModel), isList: true, prop: 'packageManifests' },
+        {
+          kind: referenceForModel(SubscriptionModel),
+          isList: true,
+          prop: 'subscriptions',
+          namespace,
```

Comment:
This doesn't look correct. The subscription and install plan are not always in the same namespace depending on the install mode of the operator.

Reply:
You're right. I just tested it out and I found a bug because of this issue in the subscriptions page. I will update the PR>